sspl_test/alerts/realstor/test_real_stor_dg_sensor.py

```diff
@@ -0,0 +1,141 @@
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+# -*- coding: utf-8 -*-
+import json
+import os
+import psutil
+import time
+import sys
+
+from default import world
+from messaging.ingress_processor_tests import IngressProcessorTests
+from messaging.egress_processor_tests import EgressProcessorTests
+
+
+def init(args):
+    pass
+
+def test_real_stor_dg_sensor(agrs):
+    check_sspl_ll_is_running()
+    dg_sensor_message_request("enclosure:cortx:disk_group")
+    dg_sensor_msg = None
+    time.sleep(4)
+    while not world.sspl_modules[IngressProcessorTests.name()]._is_my_msgQ_empty():
+        ingressMsg = world.sspl_modules[IngressProcessorTests.name()]._read_my_msgQ()
+        print("Received: %s" % ingressMsg)
+        try:
+            # Make sure we get back the message type that matches the request
+            msg_type = ingressMsg.get("sensor_response_type")
+            time.sleep(0.1)
+            if msg_type['info']['resource_type'] == "enclosure:cortx:disk_group":
+                dg_sensor_msg = msg_type
+                break
+        except Exception as exception:
+            time.sleep(0.1)
+            print(exception)
```

Comment:
we already have a function `sspl_test/common.py:get_fru_response()` to do this in a single line.

Reply:
Done